pkg/config/runtime-manifest.go

```diff
@@ -147,43 +189,85 @@ func (m *RuntimeManifest) buildSourceData() (map[string]interface{}, error) {
 		m.sensitiveValues = append(m.sensitiveValues, val)
 		creds[pe] = val
 	}
-	bundle["outputs"] = m.outputs
+
+	bun["outputs"] = m.outputs
 	for _, output := range m.outputs {
 		m.sensitiveValues = append(m.sensitiveValues, output)
 	}
+
 	deps := make(map[string]interface{})
-	bundle["dependencies"] = deps
-	for name, dependency := range m.Dependencies {
-		dep, sensitives, err := dependency.resolve()
-		if err != nil {
-			return nil, err
+	bun["dependencies"] = deps
+	for alias, bun := range m.bundles {
+		// TODO: Do we still want to allow resolving against bundle.dependencies.ALIAS.parameters.NAME and credentials?
```

Comment:
We aren't going to support bundle.dependencies.DEP.credentials anymore. I will come back in another PR and remove that from the docs and examples.